packages/tools/bundleDefaultPlugins.ts

```diff
@@ -94,3 +94,12 @@ if (require.main === module) {
 		process.exit(1);
 	});
 }
+
+module.exports = async function bundleDefaultPlugins() {
+	try {
+		await start();
+	}	catch (e) {
+		console.error('Error bundling default plugins ', e);
+		process.exit(1);
+	}
+};
```

Comment:
All that indirection shouldn't be needed. If the script fails, gulp handles the error